src/modules/users/users.query.js

```diff
@@ -103,6 +105,24 @@ const deleteRelQuery = (body) => {
     return query;
 };
 
+// Other Queries
+const googleSignUpQuery = (uuid, body) => {
+    const query = `
+        CREATE (u:Student:User 
```

Comment:
we could have a property with the google api results as an OBJ to store data related to the Google API. 
Or we could just leave it with null as default